OSRMTextInstructions.podspec

```diff
@@ -48,10 +48,10 @@ Pod::Spec.new do |s|
 
   s.dependency "MapboxDirections.swift"
 
+  s.prepare_command = "./json2plist.sh"
```

Comment:
The equivalent for Carthage is the Run Script build phase included in each framework target that runs json2plist.sh.